.github/workflows/ci-matrix-build.yml

```diff
@@ -0,0 +1,91 @@
+#This workflow runs matrix build with both versions of setup MATLAB action hourly
+
+name: MATLAB matrix build
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [main, setup-matlab-matrix-build]
+  pull_request:
+  # Schedule the workflow to run hourly
+  schedule:
+    - cron: "0 * * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Since we can't use cron and we're merging into the hourly branch now, this can probably just be reduced to something like:

```yaml
# Controls when the action will run. 
on:
  push:
  pull_request:
  workflow_dispatch:
```